lighthouse-core/lib/url-shim.js

```diff
@@ -215,6 +215,30 @@ class URLShim extends URL {
     const urlScheme = protocol.includes(':') ? protocol.slice(0, protocol.indexOf(':')) : protocol;
     return NON_NETWORK_SCHEMES.includes(urlScheme);
   }
+
+  /**
+   * @param {string} src
+   * @return {string|undefined}
+   */
+  static guessMimeType(src) {
+    try {
```

Comment:
which part here can throw? Just the new URL, right? can we reduce the try/catch to just that part?